pkg/querier/queryrange/roundtrip_test.go

```diff
@@ -214,7 +234,7 @@ func TestInstantQueryTripperware(t *testing.T) {
 	lreq := &LokiInstantRequest{
 		Query:     `sum by (job) (bytes_rate({cluster="dev-us-central-0"}[15m]))`,
 		Limit:     1000,
-		Direction: logproto.FORWARD,
+		Direction: logproto.BACKWARD,
```

Comment:
nit: Why change this to backward (metric queries don't really have this concept)

Reply:
I made this trying to figure out why the return type was different. Will revert this.